x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_advanced_fields.tsx

```diff
@@ -0,0 +1,327 @@
+              defaultMessage: 'Optional description',
+            })}
+          />
+        </EuiFormRow>
+      </EuiDescribedFormGroup>
+      <EuiDescribedFormGroup
+        title={
+          <h4>
+            <FormattedMessage
+              id="xpack.fleet.agentPolicyForm.namespaceFieldLabel"
+              defaultMessage="Default namespace"
+            />
+          </h4>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.fleet.agentPolicyForm.namespaceFieldDescription"
+            defaultMessage="Namespaces are a user-configurable arbitrary grouping that makes it easier to search for data and manage user permissions. A policy namespace is used to name its integration's data streams. {fleetUserGuide}."
+            values={{
+              fleetUserGuide: (
+                <EuiLink href={docLinks.links.fleet.datastreamsNamingScheme} target="_blank">
+                  {i18n.translate(
+                    'xpack.fleet.agentPolicyForm.nameSpaceFieldDescription.fleetUserGuideLabel',
+                    { defaultMessage: 'Learn more' }
+                  )}
+                </EuiLink>
+              ),
+            }}
+          />
+        }
+      >
+        <EuiFormRow
+          fullWidth
+          error={touchedFields.namespace && validation.namespace ? validation.namespace : null}
+          isInvalid={Boolean(touchedFields.namespace && validation.namespace)}
+        >
+          <EuiComboBox
+            fullWidth
+            singleSelection
+            noSuggestions
+            selectedOptions={agentPolicy.namespace ? [{ label: agentPolicy.namespace }] : []}
+            onCreateOption={(value: string) => {
+              updateAgentPolicy({ namespace: value });
+            }}
+            onChange={(selectedOptions) => {
+              updateAgentPolicy({
+                namespace: (selectedOptions.length ? selectedOptions[0] : '') as string,
+              });
+            }}
+            isInvalid={Boolean(touchedFields.namespace && validation.namespace)}
+            onBlur={() => setTouchedFields({ ...touchedFields, namespace: true })}
+          />
+        </EuiFormRow>
+      </EuiDescribedFormGroup>
+      <EuiDescribedFormGroup
+        title={
+          <h4>
+            <FormattedMessage
+              id="xpack.fleet.agentPolicyForm.monitoringLabel"
+              defaultMessage="Agent monitoring"
+            />
+          </h4>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.fleet.agentPolicyForm.monitoringDescription"
+            defaultMessage="Collecting monitoring logs and metrics will also create an {agent} integration. Monitoring data will be written to the default namespace specified above."
+            values={{
+              agent: (
+                <AgentPolicyPackageBadge pkgName={'elastic_agent'} pkgTitle={'Elastic Agent'} />
```

Comment:
currently displaying a generic badge. in order to display the package's real badge, I would need to know the latest version number, which seems overkill